app/workers/activitypub/delivery_worker.rb

```diff
@@ -38,9 +39,15 @@ def build_request(http_client)
     Request.new(:post, @inbox_url, body: @json, http_client: http_client).tap do |request|
       request.on_behalf_of(@source_account, :uri, sign_with: @options[:sign_with])
       request.add_headers(HEADERS)
+      request.add_headers({ 'X-AS-Collection-Synchronization' => synchronization_header }) if ENV['DISABLE_FOLLOWERS_SYNCHRONIZATION'] != 'true' && @options[:synchronize_followers]
     end
   end
 
+  def synchronization_header
+    url_prefix = @inbox_url[/http(s?):\/\/[^\/]+\//]
```

Comment:
I guess this is the `inbox_url_prefix` and could be extracted to a method as such